src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -88,12 +88,13 @@
 (defmacro qc-and-report-exception
   [final-reports num-tests-or-options bindings & body]
   `(report-exception-or-shrunk
-     (let [num-tests-or-options# ~num-tests-or-options]
+     (let [num-tests-or-options# ~num-tests-or-options
+           final-reports# ~final-reports]
        (apply tc/quick-check
          (times num-tests-or-options#)
          (prop/for-all ~bindings
            (let [reports# (capture-reports ~@body)]
-             (swap! ~final-reports save-to-final-reports reports#)
+             (swap! final-reports# save-to-final-reports reports#)
```

Comment:
nice

Reply:
Hah, unintentional bugfix. I changed it just for aethetics but I never realized there was `for-all` wrapping it :)